docs/howto/kubernetes/gardener/kubectl.md

```diff
@@ -3,40 +3,65 @@ description: How to fetch, verify, and use your kubeconfig with kubectl.
 ---
 # Managing a Kubernetes cluster
 
-Once you [have launched a new cluster](create-shoot-cluster.md), you can interact with it using `kubectl` and a [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) file.
+Once you [have launched a new cluster](create-shoot-cluster.md), you can interact with it using `kubectl` with a [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig) file.
```

Comment:
Dropping the trailing slash on a link is considered bad form, because it subjects the user to one unnecessary redirect.

```suggestion
Once you [have launched a new cluster](create-shoot-cluster.md), you can interact with it using `kubectl` with a [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) file.
```

Reply:
Argh, I never thought of that :/